docs/app/pages/components/sections/sliders/sliders/sliders.component.html

```diff
@@ -0,0 +1,326 @@
+        </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+
+        <div class="col-md-12">
+            <h4 class="text-black p-t-md p-b-md">Range Slider with Text Inputs</h4>
+        </div>
+
+        <div class="col-md-12">
+            <ux-slider [value]="slider7.value" (valueChange)="updateValue($event)" [options]="slider7.options"></ux-slider>
+        </div>
+    </div>
+
+    <div class="row m-t-sm p-t-xs">
+        <div class="col-sm-2">
+            <input type="text" class="form-control m-b" [ngModel]="lowerValue | async" (ngModelChange)="lowerValue.next($event)">
+        </div>
+        <div class="col-sm-offset-8 col-sm-2">
+            <input type="text" class="form-control m-b" [ngModel]="upperValue | async" (ngModelChange)="upperValue.next($event)">
+        </div>
+    </div>
+
+</div>
+
+<hr>
+
+<p>The slider component can be used to allow a user to select a single value or a range of values. It can easily be customized
+    to match the desired appearance and behavior.</p>
+
+<h4>Slider Component</h4>
+
+<p>The component can be added to a page by using the following HTML:</p>
+
+<uxd-snippet [content]="snippets.compiled.sample1Html"></uxd-snippet>
+
+<blockquote>
+    <p>Reference Plunker for the code used in all examples above</p>
+</blockquote>
+
+<h4>Values</h4>
+
+<p>The value or range should be specified by adding an <code>value</code> attribute to the component tag. This should reference
+    either a single number or a <code>SliderValue</code> object, depending on the slider type specified.</p>
+
+<p>For single value selection, an example value might be:</p>
+
+<uxd-snippet [content]="snippets.compiled.sample1Ts" language="javascript"></uxd-snippet>
+
+<p>For range selection, an example value might be:</p>
+
+<uxd-snippet [content]="snippets.compiled.sample2Ts" language="javascript"></uxd-snippet>
+
+<p>The variable containing the value can be updated with the correct value, or values, when the slider is dragged is two way
+    binding is used.</p>
+
+<h4>Options</h4>
+
+<p>A wide range of options can used to customize the appearance and behavior of the component. These options should be stored
+    in an object and passed to the component by adding an <code>options</code> attribute to the directive.</p>
+
+<p>The default options for each chart are as follows:</p>
+
+<uxd-snippet [content]="snippets.compiled.sample3Ts" language="javascript"></uxd-snippet>
+
+<p>Full details on each property and the possible values for each can be found below. All properties are optional, if they are
+    not specified the default value will be used.</p>
+
+<div class="demo-attributes table-responsive">
+    <table class="table">
+        <tr>
+            <th>Property</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td class="attribute">type</td>
+            <td>Defines whether the slider should allow selection of a single value or a range of values. The two valid values
```

Comment:
Can you list the types of each property in these tables? If you break down the SliderOptions type you could also list e.g. SliderHandleOptions here and then name the type in the subsequent section.